service/controller/v19/templates/cloudformation/guest/iam_policies.go

```diff
@@ -114,6 +114,14 @@ const IAMPolicies = `{{define "iam_policies"}}
               - "ecr:ListImages"
               - "ecr:BatchGetImage"
             Resource: "*"
+
+          - Effect: "Allow"
+            Action:
+              - "autoscaling:DescribeAutoScalingGroups"
+              - "autoscaling:DescribeAutoScalingInstances"
+              - "autoscaling:SetDesiredCapacity"
+              - "autoscaling:TerminateInstanceInAutoScalingGroup"
+            Resource: "*"
```

Comment:
```suggestion
              - "autoscaling:autoscaling:DescribeTags"              
            Resource: "*"
            
          - Effect: "Allow"
            Action:
              - "autoscaling:SetDesiredCapacity"
              - "autoscaling:TerminateInstanceInAutoScalingGroup"
            Resource: "*"
            Condition: 
              StringEquals:   
                aws:RequestTag/giantswarm.io/cluster: "{{ $v.ClusterID }}" 
```
We still need to add the `ClusterID` in the iam policy adapter. I can do push a commit with that tomorrow
I would like to nail it down so only cluster ASG can be touched?